### PR TITLE
Remove SB parent (redhat)

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -44,9 +57,87 @@
...
+                <goal>verify</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+      </plugins>
+    </pluginManagement>
   </build>
 
+  <properties>
```
Comment (gytis) [inline pom.xml]: Normally properties, licenses, dependnecyManagement are defined at the top.

Comment (geoand) [reply pom.xml]: A good case for a later manual cleanup :)

Comment (gytis) [reply pom.xml]: Maybe just list all these things to the JIRA you've created yesterday?

Comment (geoand) [reply pom.xml]: Sure, I'll do that

**Pull request merged**